services/user.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = get_user_model().objects.create_user(username=username)
```

Comment:
According to the checklist, you should use double quotes in your code. Please, replace single quotes with double quotes.